f600233d1b154d507b782f84604bcb12/update/sys_script_include_5d76f38e970611106fa8b4b3f153afc4.xml

```diff
@@ -32,7 +32,9 @@ Request.prototype = {
 		request.setEndpoint(req.endpoint);
 		request.setHttpMethod(req.method);
 		request.setLogLevel("all");
-		request.setMutualAuth(req.mutualAuth || this.JKS_443);
+		if (req.mutualAuth) {
```

Comment:
This is reasonable but we also need to make sure that calling Flows were not relying on the default value that was removed (`this.JKS_443`).

Reply:
Yeah, thought of a better way overnight to maintain existing functionality.